### ISSUE TITLE
Rich text tool to add phone numbers that meet Section 508 and VA style

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#6268]
## User story
As a VAMC editor, I want share phone numbers in rich text in an easy, consistent way that meets Section 508 and VA style guidelines at https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers.

## Issue statement
The rich text editor doesn't have a systemic way to add a phone number that meets web accessibility, VA style, and mobile friendly guidelines. It would be helpful to have a way to "add phone number" so that editors can enter the numbers without having to edit HTML.

**Comment by kevwalsh** [comment]
This is a dupe of #1964.
